socrates/lib/registration/index.js

```diff
@@ -38,7 +41,8 @@ function roomOptions() {
       two: 2 * base + 2 * day,
       three: 3 * base + 2 * day,
       threePlus: 3 * base + 3 * day,
-      four: 4 * base + 3 * day
+      four: 4 * base + 3 * day,
+      displayAsBookable: (activity.isAlreadyRegistered(memberId) || !isRegistrationOpen() || activity.resourceNamed(id).canSubscribe())
```

Comment:
Ja, die Namensgebung ist mir mal wieder schwergefallen...
-> Es zeigt die Zeile als Checkboxen an. Wenn es false ist, wird die Zeile als "Sorry, this option is not available anymore " angezeigt (siehe unten)